pyani_plus/public_cli.py

```diff
@@ -870,5 +873,60 @@ def plot_run(  # noqa: C901
     return 0
 
 
+@app.command()
+def classify(
+    database: REQ_ARG_TYPE_DATABASE,
+    outdir: REQ_ARG_TYPE_OUTDIR,
+    covearge_edges: OPT_ARG_TYPE_COVERAGE_EDGES = "min",  # Default to "min"
+    identity_edges: OPT_ARG_TYPE_IDENTITY_EDGES = "mean",  # Default to "mean"
+    run_id: OPT_ARG_TYPE_RUN_ID = None,
+) -> int:
+    """Classify genomes into clusters based on ANI results."""
+    if not outdir.is_dir():
+        msg = f"ERROR: Output directory {outdir} does not exist"
+        sys.exit(msg)
+
+    if database == ":memory:" or not Path(database).is_file():
+        msg = f"ERROR: Database {database} does not exist"
+        sys.exit(msg)
+
+    session = db_orm.connect_to_db(database)
+    run = db_orm.load_run(session, run_id, check_complete=True)
+    if run_id is None:
+        run_id = run.run_id
+        print(f"INFO: Exporting run-id {run_id}")
+
+    mapping = {_.genome_hash: Path(_.fasta_filename).stem for _ in run.fasta_hashes}
+
+    identity = run.identities
+    if identity is not None:
+        identity.rename(index=mapping, columns=mapping, inplace=True)  # noqa: PD002
+    cov = run.cov_query
+    if cov is not None:
+        cov.rename(index=mapping, columns=mapping, inplace=True)  # noqa: PD002
```

Comment:
I suggest dropping your custom renaming and use the same code as export-runs and plot (on both matrices). You can add the same ``--label`` command line option.